jupyter-org-client.el

```diff
@@ -868,6 +881,12 @@ the width or height of the image."
             (base64-decode-region (point-min) (point-max))))))
     (cl-destructuring-bind (&key width height &allow-other-keys)
         metadata
+      (if (and (null width)
+               jupyter-org-adjust-image-size
```

Comment:
Can you mention this behavior in the docstring. You can just reference the documentation of `jupyter-org-adjust-image-size` somewhere in the last paragraph.